lib/seahorn/BvOpSem2.cc

```diff
@@ -157,7 +159,6 @@ struct OpSemVisitorBase {
       : m_ctx(ctx), m_efac(m_ctx.getExprFactory()), m_sem(sem) {
 
     trueE = m_ctx.m_trueE;
-    falseE = m_ctx.m_falseE;
```

Comment:
why?

Reply:
I think this was deleted by mistake, and the place where the line is moved is dead code.

Reply:
yes.. this is a mix of editor clang-format/misbehaving and my eyeballs not catching it.